woocommerce/woocommerce-ios
# Remove override of navigationbar style for LoginViewController

Closes #3018 

The warning is issued because LoginViewController+Woo overrides `preferredStatusBarStyle`.

The override is accompanied by this comment:

```
    /// TODO: Create a property in Authenticator then delete this
    /// https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/26
```

And it turns out the referenced issue wordpress-mobile/WordPressAuthenticator-iOS#26 has already been closed. So it should be possible to remove the override on our side.

| Before | After |
| ---- | ---- |
| <img src="https://user-images.githubusercontent.com/2722505/98171309-7cee1f80-1f2a-11eb-889e-53fc7e4fe6fa.png" width="350"/> | <img src="https://user-images.githubusercontent.com/2722505/98171298-76f83e80-1f2a-11eb-9671-b6e77cac8428.png" width="350"/> |

# What's new
* Remove LoginViewcontroller+Woo
* Update the status bar style provided to the authenticator style to `lightContent`, as WPAuthenticator would take and apply this style.

# How to test
* Checkout the branch, build and run
* Log out if necessary, navigate to the login flow.
* Check the status bar style is still light.

Update release notes:
